### ISSUE TITLE
Capturing reaction names into Reaction metadata

### ISSUE THREAD

**Issue by oxinabox** [sebapersson/SBMLImporter.jl#61]
Consider a reaction like
```
<reaction metaid="_704943" id="vPTS" name="Phosphotransferase system" reversible="false">
...
```

It would be useful to capture the `id` and `name` somewhere.
I think it is legal in Catalyst to put them in the metadata for the Reaction.
Then when e.g. modifing the system programatically, or otherwise examing it, it would be available to query on

**Comment by sebapersson** [comment]
This is a good suggestion!

After looking a bit at Catalyst, I think it would make most sense to add this information in the `description` field for a `Catalyst.Reaction`, maybe like:

```julia
description = "Id: $id. Name $name
```

(seems data can only be added a string in the `description` field).

Does this sound good (maybe putting this in the `description` field is not optimal if you want to query based on the `id` and `name`)?

@TorkelE are there any other meta-data field where it would be relevant to add this information?

**Comment by TorkelE** [comment]
Currently only misc and description and noise_scaling are officially demonstrated.

In practice you can give whatever and then get metadata(rx, :field), see how our fields are implemented.

We can also add new ones, it is really easy

**Comment by sebapersson** [comment]
Thanks, metadata field was more flexible than I thought.

Then I will add in the `metadata` a `name` and `id` entry that can be retrieved via:

```julia
getmetadata(r, :name)
getmetadata(r, :id)
```
if that sounds good @oxinabox ?

Maybe also later adding a `name` field would be relevant for Catalyst to have in the API with something like a `getname` function.

**Comment by TorkelE** [comment]
Yes. Right now we dont really show the full flexibility on reaction metadata (to make it more similar to symbolic variables metadata, which has some heavy restrictions).

I plan on adding a specific dev doc section on these though.

**Comment by oxinabox** [comment]
Yes, I think the suggestion of putting both in metadata as seperate fields would be ideal.

**Comment by isaacsas** [comment]
The main thing is to use the `getmetadata` interface as the internal representation within a `reaction` for storing metadata (currently a vector) could change.

**Comment by isaacsas** [comment]
I should mention that this is new thanks to @TorkelE, and we'd be open to suggestions for making it more useful going forward.

**Comment by sebapersson** [comment]
This has now been fixed by #63. As soon as I get the next minor release updated reaction `id` and `name` are obtained via:

```julia
using SBMLImporter, Catalyst
prn, cb = load_SBML(path_SBML)
sbml_reactions = reactions(prn.rn)
getmetadata(sbml_reactions[1], :id)
getmetadata(sbml_reactions[1], :name)
```
